mlxtend/frequent_patterns/apriori.py

```diff
@@ -105,6 +111,19 @@ def apriori(df, min_support=0.5, use_colnames=False, max_len=None, verbose=0):
     http://rasbt.github.io/mlxtend/user_guide/frequent_patterns/apriori/
 
     """
+
+    def _support(_x, _n_rows, _is_sparse):
```

Comment:
Not really important since private functions don't show up in the docs, but usually, we use the NumPy-style documentation style. That's because I had written a Python doc to markdown parser that automatically generates the documentation for the HTML website documentation. No need to change it here though because like I said, it won't appear on the website

Reply:
Well, this is definitely the time to change it if you want me to. I'm happy to do it to conform to the overall style.

Reply:
In fact, if you don't mind, let me just do that real quick...even if it doesn't appear on the website we should be consistent to the overall style. Why build in inconsistencies?

Reply:
Done

Reply:
ok sure, please go ahead then. Thanks!